source/includes/documentation/_vue_js_elements.md

```diff
@@ -4247,107 +4247,203 @@ v-parallax element has following methods:
 
 ### 5.36 TreeView
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/treeview/)
+[Treeview overview](https://v2.vuetifyjs.com/en/components/treeview/)
 
 __Java__:
 - _com.epam.jdi.light.vuetify.elements.complex.TreeView.java_
+- _com.epam.jdi.light.vuetify.elements.complex.TreeViewNode.java_
 
 You can specify locators for the TreeView by using `JDITreeView` annotation:
 
-|Locator in `JDITreeView` | Description
---- | --- 
-`core` | used to set the _core_ of the TreeView
-`coreNodes` | used to set _nodes_ in the _core_ of TreeView
-`nodeNodes` | used to set _nodes_ in the _node_ of TreeView
-`root` | used set element with data in TreeView node
-`toggle` | used to set _toggle_ in the _root_ of TreeView
-`checkbox` | used to set _checkbox_ in the _root_ of TreeView
-`content` | used to set _content_ in the _root_ of TreeView
-`full` | used to set class that used as a sign of _full_ checked checkbox
-`part` | used to set class that used as a sign of _partly_ checked checkbox
-`not` | used to set class that used as a sign of _not_ checked checkbox
+| Locator in `JDITreeView` | Description                                                        
+--------------------------|--------------------------------------------------------------------
+ `core`                   | used to set the _core_ of the TreeView                             
+ `full`                   | used to set class that used as a sign of _full_ checked checkbox   
+ `part`                   | used to set class that used as a sign of _partly_ checked checkbox 
+ `not`                    | used to set class that used as a sign of _not_ checked checkbox    
```

Comment:
why did you choose only 4? just as an example? :)

Reply:
discussed it with Vladimir and took them back